gdi/opentelemetry/automatic-discovery/k8s/k8s-java-traces-tutorial/config-k8s-for-java.rst

```diff
@@ -55,7 +55,10 @@ Now, you need to configure Helm to correctly install the Splunk Distribution of
           - Tags data that the application sends to Splunk Observability Cloud, allowing you to see the data in Splunk APM
         * - ``certmanager.enabled``
           - ``true``
-          - Activates the certification manager for Helm
```

Comment:
I think the placement of description has been changed here. The last item should be the description? We don't want to remove this line for certmanager